### DISCUSSION TITLE
failed get storage: can't find storage with rawPath: /

### DISCUSSION THREAD

**Discussion by KimmyLueng** [AlistGo/alist#1654] [Q&A]
failed get storage: can't find storage with rawPath: /

无论是docker还是命令行安装，安装完成后进入web端都是这样提示

**Comment by jile1997** [comment]
我也遇到了这个问题 log里面有一句报错 failed to init aria2 client: Post "http://localhost:6800/jsonrpc": dial tcp 127.0.0.1:6800: connect: connection refused

failed get aria2 version

**Reply by jile1997** [reply]
找到说明了 可以看下这个链接 https://alist.nn.ci/zh/faq/howto.html

**Reply by Rubmaps** [reply]
想问一下解决了吗？没看懂这个链接，请教一下该怎么做呢？

**Reply by yh6441** [reply]
> 想问一下解决了吗？没看懂这个链接，请教一下该怎么做呢？

页面底部点登陆，在管理里添加存储就可以了

**Reply by ego8898** [reply]
请问，登陆的用户名和密码是多少？

**Reply by KimmyLueng** [reply]
> 请问，登陆的用户名和密码是多少？
上面链接里有说明

**Reply by caizili999** [reply]
怎么解决吗

**Reply by niya99** [reply]
> 请问，登陆的用户名和密码是多少？
> 上面链接里有说明

你好，我是小白刚接触，没找到说明文档呀，可以请您指个路吗？

**Reply by kmoonn** [reply]
$查看管理员信息，请执行
cd /home/lighthouse/alist
./alist admin

**Comment by phoenix13023** [comment]
我也是新用户，遇到这个问题。
推荐 看看这页 引导吧：https://segmentfault.com/a/1190000043031147 

一直以为是我安装出现了错误。。。。没想到默认就是这样，算是安装正确的显示。。。。。

**Reply by gaorufeng** [reply]
查看到admin密码,但是用这个密码登录提示密码不对,进不了管理
